package.json

```diff
@@ -59,6 +60,7 @@
     "vue-chartjs": "^3.4.2",
     "vue-class-component": "^7.2.3",
     "vue-i18n": "^8.11.2",
+    "vue-material-design-icons": "^5.1.2",
```

Comment:
could you remove this module?
If you want to use icon, basically choose it from already using module:
https://v15.vuetifyjs.com/en/components/icons

if you have specific reason to use another icon set, please tell us.

Reply:
Done